index.js

```diff
@@ -92,23 +92,22 @@ form.addEventListener('submit', (event) => {
 
 function addBookToDOM(book) {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.